packages/suite-analytics/src/types/definitions.ts

```diff
@@ -4,7 +4,7 @@ export type OnboardingAnalytics = {
     startTime: number;
     firmware: 'install' | 'update' | 'skip' | 'up-to-date';
     seed: 'create' | 'recovery' | 'recovery-in-progress';
-    seedType: 'standard' | 'shamir';
+    seedType: 'shamir-default' | 'shamir-advance' | '12-words' | '24-words';
```

Comment:
Can I do this? Who I shall notify about this?

Reply:
There is analytics page in Notion

Reply:
Please update the possible values [here](https://www.notion.so/satoshilabs/seedType-049a62397d61466f8150f8b9f9dded2a?pvs=4) when merged

Reply:
done